vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridSetItemsAfterDetachIT.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.grid.it;
+
+import com.vaadin.flow.component.grid.testbench.GridElement;
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import org.junit.Assert;
+import org.junit.Test;
+
+@TestPath("gridsetitemsafterdetachpage")
+public class GridSetItemsAfterDetachIT extends AbstractComponentIT {
+
+    @Test
+    public void reattachGrid() {
```

Comment:
For me, the test produced a false positive result after reverting the fix. Tested also by interacting manually with the page.
@alvarezguille can you test as well?

Reply:
Nevermind. The error was suppressed because of production mode.